test/validation/api/ipsec/ipsec.c

```diff
@@ -378,8 +378,15 @@ void ipsec_sa_param_fill(odp_ipsec_sa_param_t *param,
 	if (auth_key)
 		param->crypto.auth_key = *auth_key;
 
-	if (extra_key)
-		param->crypto.cipher_key_extra = *extra_key;
```

Comment:
I think it would be more logical to introduce `extra_cipher_key`/`extra_auth_key` here.

Reply:
This was minimal change to test cases, but agree test cases are easier to read when those are separate params. Done in v2.